src/styles/card.scss

```diff
@@ -0,0 +1,84 @@
+.card {
+  font-family: Roboto, Arial, sans-serif;
+  width: 198px;
+  height: 406px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__body {
+    padding: 32px 16px 16px;
+    width: 166px;
+    height: 134px;
+  }
+
+  &__text {
+    width: 100%;
+    height: 36px;
+    font-family: Roboto, Arial, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__text--price {
+    font-weight: 400;
+    margin-top: 24px;
+    height: 18px;
```

Comment:
```suggestion

```
The height of the element should depend on the content and shouldn't be hardcoded